From 6e8ea807f33ee675cd6d79df6aaad9e94e72cbc6 Mon Sep 17 00:00:00 2001
From: Limeng <Meng.Li@windriver.com>
Date: Mon, 21 May 2018 08:46:34 +0000
Subject: [PATCH 24/25] arch: arm: dts: zc702 enable watchdog node and add
 reset-on-timeout property

enable watchdog and reset-on-timeout

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm/boot/dts/zynq-zc702-adv7511.dts | 5 +++++
 arch/arm/boot/dts/zynq-zc702.dts         | 5 +++++
 2 files changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/zynq-zc702-adv7511.dts b/arch/arm/boot/dts/zynq-zc702-adv7511.dts
index ba2251bee602..e7cc4303a1ab 100644
--- a/arch/arm/boot/dts/zynq-zc702-adv7511.dts
+++ b/arch/arm/boot/dts/zynq-zc702-adv7511.dts
@@ -647,3 +647,8 @@
 	pinctrl-0 = <&pinctrl_usb0_default>;
 	xlnx,phy-reset-gpio = <&gpio0 7 0>;
 };
+
+&watchdog0 {
+	status = "okay";
+	reset-on-timeout;
+};
diff --git a/arch/arm/boot/dts/zynq-zc702.dts b/arch/arm/boot/dts/zynq-zc702.dts
index 8e47476dba30..3dff01bc9d36 100644
--- a/arch/arm/boot/dts/zynq-zc702.dts
+++ b/arch/arm/boot/dts/zynq-zc702.dts
@@ -460,3 +460,8 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usb0_default>;
 };
+
+&watchdog0 {
+	status = "okay";
+	reset-on-timeout;
+};
-- 
2.17.0

